app/assets/javascripts/templates/directives/markdown_input.jst.ejs

```diff
@@ -0,0 +1,6 @@
+<div class="text-box">
+  <textarea class="atwho" ng-model="content" placeholder="{{ filler }}"></textarea>
```

Comment:
yeah, content is just `$scope.content` and thats whatever is passed into the directive